RaspberryPiDeployment.md

```diff
@@ -142,23 +142,31 @@ sudo mkdir /mnt/usb/docker
 sudo ln -s /mnt/usb/docker /var/lib/docker
 ```
 
-**Step 13** - Move Swapfile to USB and increase size
+**Step 13** - Move Swapfile to USB and increase size.  Skip this step if using RPi4 with more than 1 GB RAM.
 ```bash 
 sudo nano /etc/dphys-swapfile
 ```
 Change the CONF_SWAPFILE line to 
 CONF_SWAPFILE=/mnt/usb/swapfile
 
 Change the CONF_SWAPSIZE line to 
-CONF_SWAPSIZE=2048
+CONF_SWAPSIZE=1024
 
 Stop and restart the swapfile service
 ```bash
 sudo /etc/init.d/dphys-swapfile stop
 sudo /etc/init.d/dphys-swapfile start
 ```
 
-**Step 14** - Install BTCPayServer.  
+**Step 14** - Install Docker. 
+Docker-CE is not available yet for Raspbian Buster.  Until Docker is available in the official repo it has to be installed manually. 
+```bash
+curl -fsSL https://get.docker.com|sed -e 's/buster/stretch/' > get-docker.sh
```

Comment:
It's patched in the setup script so don't need to update the installation docs with the patch https://github.com/btcpayserver/btcpayserver-docker/pull/176